packages/backend/src/apiV1.ts

```diff
@@ -89,6 +90,7 @@ apiV1Router.post('/login', passport.authenticate('local'), (req, res, next) => {
         } else {
             res.json({
                 status: 'ok',
+                results: req.user!,
```

Comment:
we should never expose/return the `req.user` since it is of type `SessionUser` instead of `LightdashUser`. 
The `SessionUser` type might have sensitive information like the userId.

Reply:
TY!